.github/workflows/daily_ci.yml

```diff
@@ -6,53 +6,68 @@ on:
     - cron: "00 15 * * 1-5"
 
 jobs:
+  getVersion:
+    uses: ./.github/workflows/dafny_version.yaml
+  getVerifyVersion:
+    uses: ./.github/workflows/dafny_verify_version.yaml
+  # not yet used, because we don't actually CI formatting in this project?!?
+  getFormatVersion:
+    uses: ./.github/workflows/dafny_format_version.yaml
```

Comment:
Blocking: `Version` is vague; we have dependency versions, our product versions, Java/.NET/Python runtime versions, etc.

Name these `dafnyRuntimeVersion`, `dafnyVerifyVersion`, & `dafnyFormantVersion`.

I know this may seem excessive, but I am not very smart and get confused all the time.
I need help keeping everything straight.